db/seeds.d/020-permissions_list.rb

```diff
@@ -61,6 +61,7 @@ def permissions
         ['Host', 'console_hosts'],
         ['Host', 'ipmi_boot_hosts'],
         ['Host', 'forget_status_hosts'],
+        ['Host', 'edit_instance_hosts'],
```

Comment:
As discussed during the meeting, being able to maintain an instance probably gives you the permissions to become full admin. That means I wonder if we need this permission at all.

Reply:
Done